modules/serverless-rn-1-7-0.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-7-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.7.0
+
+[id="new-features-1-7-0_{context}"]
+== New features
+* {ServerlessProductName} 1.7.0 is now Generally Available (GA) on {product-title} 4.3 and newer versions. In previous versions, {ServerlessProductName} was a Technology Preview.
+* {ServerlessProductName} now uses Knative Serving 0.13.1.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.1.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.1.
+* Knative Eventing is now available as a Technology Preview with {ServerlessProductName}.
+
```

Comment:
Let's add that the `trustedCA` setting in Openshift's cluster-wide proxy is now supported as well, so we're fully compatible with OCPs proxy settings.